tasks/export_abi.js

```diff
@@ -50,8 +50,10 @@ subtask(
 
     abi = abi.filter((element, index, array) => config.filter(element, index, array, fullName));
 
-    if (config.pretty) {
+    if (config.format == "minimal") {
       abi = new Interface(abi).format(FormatTypes.minimal);
+    } else if (config.format == "fullName") {
+      abi = new Interface(abi).format(FormatTypes.fullName);
     }
```

Comment:
Throw an error if `format` doesn't match any of the possible options.

Reply:
addressed 👍🏼